packages/contractkit/src/providers/celo-private-keys-subprovider.ts

```diff
@@ -112,9 +127,17 @@ export class CeloPrivateKeysWalletProvider extends PrivateKeyWalletSubprovider {
       }
     }
 
-    if (txParams.gasPrice == null) {
-      txParams.gasPrice = await this.getGasPrice()
+    if (isEmpty(txParams.gasPrice)) {
+      // TODO(ashishb): For now, this will only work when gas payments are in Celo Gold
```

Comment:
So what happens right now if contractkit consumers want to pay in cUSD? It just doesn't work?

Reply:
Obsolete comment. I wrote this earlier, so, as not to lose track of the limitation, fixed it and then forgot to remove the comment. Comment removed.